examples/demo_threading/CMakeLists.txt

```diff
@@ -46,14 +46,14 @@ if(NOT CxxQt_FOUND)
     FetchContent_Declare(
         CxxQt
         GIT_REPOSITORY https://github.com/kdab/cxx-qt-cmake.git
-        GIT_TAG main
+        GIT_TAG cmake-qt-modules
```

Comment:
We can't merge this of course, so make sure we undo that before merging :)

Reply:
done